### DISCUSSION TITLE
Is it possible to set the window size of gccontent adapter?

### DISCUSSION THREAD

**Discussion by lznbb** [GMOD/jbrowse-components#3985] [Q&A] [answered]
I'm using jbrowse2 and the GC content adapter seems set by 1000 bp for each block. In my case, it would be very nice if the size can be tuned. Is it possible or any plugin/workaround I can refer to? Thanks in advance.

**Comment by cmdcolin** [comment, marked as answer]
It is not currently, but this was definitely intended to get added. i made a PR just now to try to help here https://github.com/GMOD/jbrowse-components/pull/3986

can get this branch build with

`jbrowse create --branch gc_content_edit newdir`

let me know if that helps

**Reply by lznbb** [reply]
very helpful. Thank you so much! One minor question, I will close this ticket for now, but would it be possible to let me know after this feature is released so that I don't need to specify the branch any more?

**Reply by cmdcolin** [reply]
sure thing

**Reply by cmdcolin** [reply]
latest release v2.7.1 now released with this. I thought that maybe in the future 'auto-adjusting' window sizes could be a thing according to zoom level, but hopefully that helps for now

**Reply by cmdcolin** [reply]
let me know if you have any issues with it

**Reply by lznbb** [reply]
Sounds great! I will try in days, but so far it works well so I believe there will be no issue with the released version. Thanks a lot for the quick response
